src/gui/properties/peerlistwidget.cpp

```diff
@@ -67,7 +67,7 @@ PeerListWidget::PeerListWidget(PropertiesWidget *parent)
     setSelectionMode(QAbstractItemView::ExtendedSelection);
     header()->setStretchLastSection(false);
     // List Model
-    m_listModel = new QStandardItemModel(0, PeerListDelegate::COL_COUNT);
+    m_listModel = new QStandardItemModel(0, PeerListDelegate::COL_COUNT, this);
```

Comment:
Do you plan to squash commits? To me it makes sense to squash https://github.com/qbittorrent/qBittorrent/pull/6487/commits/eac4f21bae12898ca17e0c31106b4a30cbadd684 and https://github.com/qbittorrent/qBittorrent/pull/6487/commits/946d2d36d2b0dff0f4332bd64ffa6a1b9fa56fde